### PR TITLE
Call support with WebRTC

### PR DESCRIPTION
**Pull request by bmarty**
This PR is created to quickly see diff with develop branch

### CODE REVIEW

**Comment by BillCarsonFr** [comment]
## Outgoing call
image


## Incoming call

* When user using actively phone
image

* When screen is locked (ability to reply without entering PIN)
image


## On going call

* Laptop (landscape) <> Mobile (portrait)
image

File: vector/build.gradle [hunk 1]
```diff
@@ -390,6 +390,9 @@ dependencies {
 
     implementation 'com.github.BillCarsonFr:JsonViewer:0.5'
 
+    // TODO meant for development purposes only
```
[the remark this answers is not in the document]
Comment (bmarty) [reply vector/build.gradle:393]: As discussed IRL, we should import what have been done on Riot-Android to build WebRTC lib

[the remark this answers is not in the document]
Comment (bmarty) [reply vector/build.gradle:393]: @BillCarsonFr can you handle it please?

**Comment by rakshazi** [comment]
Holy cow, RiotX v0.23 will receive calls?!
I love you, guys!

**Comment by progserega** [comment]
RiotX  0.23.0-dev [40023001] (G-b3059) do not play sound, when occur incoming call.

**Comment by bmarty** [comment]
> RiotX 0.23.0-dev [40023001] (G-b3059) do not play sound, when occur incoming call.

Thanks for the feedback. However, you should open a new issue instead of commenting a merged PR for more visibility :)

**Pull request merged**